atomics link errors on aarch64-unknown-linux-musl

Note: not cross-compiling, building on `aarch64-unknown-linux-musl` for `aarch64-unknown-linux-musl`.

Attempting to build any crate that depends on `libgit2-sys` on `aarch64-unknown-linux-musl` gives link errors about undefined references to `__aarch64_ldadd4_acq_rel`, `__aarch64_swp8_acq_rel`, `__aarch64_cas8_acq_rel`, etc:

```
376 | = note: /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o): in function `git_attr_rule__clear':
377 | attr_file.c:(.text.git_attr_rule__clear+0x60): undefined reference to `__aarch64_ldadd4_acq_rel'
378 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o): in function `merge_assignments':
379 | attr_file.c:(.text.merge_assignments+0x24): undefined reference to `__aarch64_ldadd4_acq_rel'
380 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o): in function `git_attr_file__new':
381 | attr_file.c:(.text.git_attr_file__new+0x74): undefined reference to `__aarch64_ldadd4_acq_rel'
382 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o): in function `git_attr_file__free':
383 | attr_file.c:(.text.git_attr_file__free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
384 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o): in function `git_attr_assignment__parse':
385 | attr_file.c:(.text.git_attr_assignment__parse+0x35c): undefined reference to `__aarch64_ldadd4_acq_rel'
386 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attr_file.o):attr_file.c:(.text.git_attr_assignment__parse+0x414): more undefined references to `__aarch64_ldadd4_acq_rel' follow
387 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attrcache.o): in function `attr_cache__free.part.0':
388 | attrcache.c:(.text.attr_cache__free.part.0+0x80): undefined reference to `__aarch64_swp8_acq_rel'
389 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.attr_cache__free.part.0+0x98): undefined reference to `__aarch64_swp8_acq_rel'
390 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attrcache.o): in function `git_attr_cache__get':
391 | attrcache.c:(.text.git_attr_cache__get+0xd4): undefined reference to `__aarch64_ldadd4_acq_rel'
392 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x158): undefined reference to `__aarch64_swp8_acq_rel'
393 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x16c): undefined reference to `__aarch64_ldadd4_acq_rel'
394 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x17c): undefined reference to `__aarch64_swp8_acq_rel'
395 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x194): undefined reference to `__aarch64_swp8_acq_rel'
396 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x254): undefined reference to `__aarch64_cas8_acq_rel'
397 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: attrcache.c:(.text.git_attr_cache__get+0x278): undefined reference to `__aarch64_swp8_acq_rel'
398 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attrcache.o): in function `git_attr_cache__init':
399 | attrcache.c:(.text.git_attr_cache__init+0x13c): undefined reference to `__aarch64_cas8_acq_rel'
400 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(attrcache.o): in function `git_attr_cache_flush':
401 | attrcache.c:(.text.git_attr_cache_flush+0x14): undefined reference to `__aarch64_swp8_acq_rel'
402 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(blob.o): in function `git_blob__parse':
403 | blob.c:(.text.git_blob__parse+0x20): undefined reference to `__aarch64_ldadd4_acq_rel'
404 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config.o): in function `git_config__add_internal':
405 | config.c:(.text.git_config__add_internal+0x50): undefined reference to `__aarch64_ldadd4_acq_rel'
406 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: config.c:(.text.git_config__add_internal+0xcc): undefined reference to `__aarch64_ldadd4_acq_rel'
407 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config.o): in function `git_config_free':
408 | config.c:(.text.git_config_free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
409 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: config.c:(.text.git_config_free+0x8c): undefined reference to `__aarch64_ldadd4_acq_rel'
410 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config.o):config.c:(.text.git_config_new+0x70): more undefined references to `__aarch64_ldadd4_acq_rel' follow
411 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config_cache.o): in function `git_repository__configmap_lookup':
412 | config_cache.c:(.text.git_repository__configmap_lookup+0x9c): undefined reference to `__aarch64_cas8_acq_rel'
413 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(diff.o): in function `git_diff_free':
414 | diff.c:(.text.git_diff_free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
415 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(diff.o): in function `git_diff_addref':
416 | diff.c:(.text.git_diff_addref+0x10): undefined reference to `__aarch64_ldadd4_acq_rel'
417 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(diff_generate.o): in function `git_diff__from_iterators':
418 | diff_generate.c:(.text.git_diff__from_iterators+0xcc): undefined reference to `__aarch64_ldadd4_acq_rel'
419 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_reuc_clear':
420 | index.c:(.text.git_index_reuc_clear+0x3c): undefined reference to `__aarch64_swp8_acq_rel'
421 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_clear':
422 | index.c:(.text.git_index_clear+0x144): undefined reference to `__aarch64_swp8_acq_rel'
423 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_free':
424 | index.c:(.text.git_index_free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
425 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_open':
426 | index.c:(.text.git_index_open+0x160): undefined reference to `__aarch64_ldadd4_acq_rel'
427 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_read_tree':
428 | index.c:(.text.git_index_read_tree+0x18c): undefined reference to `__aarch64_swp8_acq_rel'
429 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_snapshot_new':
430 | index.c:(.text.git_index_snapshot_new+0x24): undefined reference to `__aarch64_ldadd4_acq_rel'
431 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: index.c:(.text.git_index_snapshot_new+0x30): undefined reference to `__aarch64_ldadd4_acq_rel'
432 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: index.c:(.text.git_index_snapshot_new+0x78): undefined reference to `__aarch64_ldadd4_acq_rel'
433 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_snapshot_release':
434 | index.c:(.text.git_index_snapshot_release+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
435 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(index.o): in function `git_index_write':
436 | index.c:(.text.git_index_write+0x94): undefined reference to `__aarch64_ldadd4_acq_rel'
437 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(mwindow.o):mwindow.c:(.text.git_mwindow_get_pack+0x8c): more undefined references to `__aarch64_ldadd4_acq_rel' follow
438 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_set_odb':
439 | repository.c:(.text.git_repository_set_odb+0x24): undefined reference to `__aarch64_swp8_acq_rel'
440 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_odb+0x3c): undefined reference to `__aarch64_ldadd4_acq_rel'
441 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_odb+0x48): undefined reference to `__aarch64_swp8_acq_rel'
442 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_odb+0x60): undefined reference to `__aarch64_swp8_acq_rel'
443 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_refdb__weakptr':
444 | repository.c:(.text.git_repository_refdb__weakptr+0x98): undefined reference to `__aarch64_swp8_acq_rel'
445 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_refdb__weakptr+0xb4): undefined reference to `__aarch64_cas8_acq_rel'
446 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_refdb__weakptr+0xd0): undefined reference to `__aarch64_swp8_acq_rel'
447 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_refdb':
448 | repository.c:(.text.git_repository_refdb+0x44): undefined reference to `__aarch64_ldadd4_acq_rel'
449 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_refdb+0xa8): undefined reference to `__aarch64_swp8_acq_rel'
450 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_refdb+0xc4): undefined reference to `__aarch64_cas8_acq_rel'
451 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_refdb+0xe0): undefined reference to `__aarch64_swp8_acq_rel'
452 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_index__weakptr':
453 | repository.c:(.text.git_repository_index__weakptr+0xd8): undefined reference to `__aarch64_swp8_acq_rel'
454 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_index__weakptr+0xf4): undefined reference to `__aarch64_cas8_acq_rel'
455 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_index__weakptr+0x110): undefined reference to `__aarch64_swp8_acq_rel'
456 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_index':
457 | repository.c:(.text.git_repository_index+0x40): undefined reference to `__aarch64_ldadd4_acq_rel'
458 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_index+0xec): undefined reference to `__aarch64_swp8_acq_rel'
459 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_index+0x108): undefined reference to `__aarch64_cas8_acq_rel'
460 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_index+0x124): undefined reference to `__aarch64_swp8_acq_rel'
461 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_set_index':
462 | repository.c:(.text.git_repository_set_index+0x20): undefined reference to `__aarch64_swp8_acq_rel'
463 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_index+0x34): undefined reference to `__aarch64_ldadd4_acq_rel'
464 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_index+0x40): undefined reference to `__aarch64_swp8_acq_rel'
465 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_set_index+0x5c): undefined reference to `__aarch64_swp8_acq_rel'
466 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_config__weakptr':
467 | repository.c:(.text.git_repository_config__weakptr+0x298): undefined reference to `__aarch64_swp8_acq_rel'
468 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_config__weakptr+0x2b0): undefined reference to `__aarch64_cas8_acq_rel'
469 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_config__weakptr+0x2c8): undefined reference to `__aarch64_swp8_acq_rel'
470 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_odb__weakptr':
471 | repository.c:(.text.git_repository_odb__weakptr+0xb8): undefined reference to `__aarch64_swp8_acq_rel'
472 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_odb__weakptr+0x10c): undefined reference to `__aarch64_cas8_acq_rel'
473 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_odb__weakptr+0x128): undefined reference to `__aarch64_swp8_acq_rel'
474 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_odb':
475 | repository.c:(.text.git_repository_odb+0x44): undefined reference to `__aarch64_ldadd4_acq_rel'
476 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_odb+0xb4): undefined reference to `__aarch64_swp8_acq_rel'
477 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_odb+0xfc): undefined reference to `__aarch64_cas8_acq_rel'
478 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_odb+0x118): undefined reference to `__aarch64_swp8_acq_rel'
479 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_config_snapshot':
480 | repository.c:(.text.git_repository_config_snapshot+0x2fc): undefined reference to `__aarch64_swp8_acq_rel'
481 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_config_snapshot+0x314): undefined reference to `__aarch64_cas8_acq_rel'
482 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_config_snapshot+0x32c): undefined reference to `__aarch64_swp8_acq_rel'
483 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository_initialbranch':
484 | repository.c:(.text.git_repository_initialbranch+0x354): undefined reference to `__aarch64_swp8_acq_rel'
485 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_initialbranch+0x36c): undefined reference to `__aarch64_cas8_acq_rel'
486 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository_initialbranch+0x384): undefined reference to `__aarch64_swp8_acq_rel'
487 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o): in function `git_repository__cleanup':
488 | repository.c:(.text.git_repository__cleanup+0x30): undefined reference to `__aarch64_swp8_acq_rel'
489 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository__cleanup+0x48): undefined reference to `__aarch64_swp8_acq_rel'
490 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository__cleanup+0x6c): undefined reference to `__aarch64_swp8_acq_rel'
491 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: repository.c:(.text.git_repository__cleanup+0x84): undefined reference to `__aarch64_swp8_acq_rel'
492 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(repository.o):repository.c:(.text.git_repository__cleanup+0xa0): more undefined references to `__aarch64_swp8_acq_rel' follow
493 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(runtime.o): in function `git_runtime_shutdown_register':
494 | runtime.c:(.text.git_runtime_shutdown_register+0x20): undefined reference to `__aarch64_ldadd4_acq_rel'
495 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: runtime.c:(.text.git_runtime_shutdown_register+0x6c): undefined reference to `__aarch64_ldadd4_acq_rel'
496 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(runtime.o): in function `git_runtime_init':
497 | runtime.c:(.text.git_runtime_init+0x38): undefined reference to `__aarch64_ldadd4_acq_rel'
498 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(submodule.o): in function `submodule_alloc':
499 | submodule.c:(.text.submodule_alloc+0x88): undefined reference to `__aarch64_ldadd4_acq_rel'
500 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(submodule.o): in function `git_submodule_free':
501 | submodule.c:(.text.git_submodule_free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
502 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(submodule.o):submodule.c:(.text.git_submodule__lookup_with_cache+0x6c): more undefined references to `__aarch64_ldadd4_acq_rel' follow
503 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `git_cache_clear':
504 | cache.c:(.text.git_cache_clear+0xe0): undefined reference to `__aarch64_ldadd8_acq_rel'
505 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `cache_store':
506 | cache.c:(.text.cache_store+0x38): undefined reference to `__aarch64_ldadd4_acq_rel'
507 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x1c0): undefined reference to `__aarch64_ldadd4_acq_rel'
508 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x20c): undefined reference to `__aarch64_ldadd8_acq_rel'
509 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x22c): undefined reference to `__aarch64_ldadd4_acq_rel'
510 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x268): undefined reference to `__aarch64_ldadd4_acq_rel'
511 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x2b0): undefined reference to `__aarch64_ldadd4_acq_rel'
512 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x2f0): undefined reference to `__aarch64_ldadd4_acq_rel'
513 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x32c): undefined reference to `__aarch64_ldadd4_acq_rel'
514 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x34c): undefined reference to `__aarch64_ldadd8_acq_rel'
515 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x3a4): undefined reference to `__aarch64_ldadd4_acq_rel'
516 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x400): undefined reference to `__aarch64_ldadd8_acq_rel'
517 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x448): undefined reference to `__aarch64_ldadd4_acq_rel'
518 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.cache_store+0x484): undefined reference to `__aarch64_ldadd4_acq_rel'
519 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `git_cache_dispose':
520 | cache.c:(.text.git_cache_dispose+0x84): undefined reference to `__aarch64_ldadd4_acq_rel'
521 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: cache.c:(.text.git_cache_dispose+0xe0): undefined reference to `__aarch64_ldadd8_acq_rel'
522 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `git_cache_get_raw':
523 | cache.c:(.text.git_cache_get_raw+0x90): undefined reference to `__aarch64_ldadd4_acq_rel'
524 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `git_cache_get_any':
525 | cache.c:(.text.git_cache_get_any+0x50): undefined reference to `__aarch64_ldadd4_acq_rel'
526 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(cache.o): in function `git_cached_obj_decref':
527 | cache.c:(.text.git_cached_obj_decref+0x18): undefined reference to `__aarch64_ldadd4_acq_rel'
528 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config_entries.o): in function `config_iterator_free':
529 | config_entries.c:(.text.config_iterator_free+0x20): undefined reference to `__aarch64_ldadd4_acq_rel'
530 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config_entries.o): in function `git_config_entries_new':
531 | config_entries.c:(.text.git_config_entries_new+0x48): undefined reference to `__aarch64_ldadd4_acq_rel'
532 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(config_entries.o):config_entries.c:(.text.git_config_entries_incref+0x10): more undefined references to `__aarch64_ldadd4_acq_rel' follow
533 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(diff_driver.o): in function `git_diff_driver_lookup':
534 | diff_driver.c:(.text.git_diff_driver_lookup+0x304): undefined reference to `__aarch64_cas8_acq_rel'
535 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: diff_driver.c:(.text.git_diff_driver_lookup+0x738): undefined reference to `__aarch64_cas8_acq_rel'
536 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o): in function `git_refdb_new':
537 | refdb.c:(.text.git_refdb_new+0x54): undefined reference to `__aarch64_ldadd4_acq_rel'
538 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o): in function `git_refdb_free':
539 | refdb.c:(.text.git_refdb_free+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
540 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o): in function `git_refdb_lookup':
541 | refdb.c:(.text.git_refdb_lookup+0x54): undefined reference to `__aarch64_ldadd4_acq_rel'
542 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o): in function `git_refdb_resolve':
543 | refdb.c:(.text.git_refdb_resolve+0x74): undefined reference to `__aarch64_ldadd4_acq_rel'
544 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o): in function `git_refdb_iterator':
545 | refdb.c:(.text.git_refdb_iterator+0x34): undefined reference to `__aarch64_ldadd4_acq_rel'
546 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/liblibgit2_sys-f3a07084e196a99e.rlib(refdb.o):refdb.c:(.text.git_refdb_iterator_next+0x28): more undefined references to `__aarch64_ldadd4_acq_rel' follow
547 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_cert.o): in function `ssl_cert_free':
548 | ssl_cert.c:(.text.ssl_cert_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
549 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_lib.o): in function `SSL_CTX_up_ref':
550 | ssl_lib.c:(.text.SSL_CTX_up_ref+0x14): undefined reference to `__aarch64_ldadd4_relax'
551 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_lib.o): in function `SSL_CTX_free':
552 | ssl_lib.c:(.text.SSL_CTX_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
553 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_lib.o): in function `SSL_free':
554 | ssl_lib.c:(.text.SSL_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
555 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_sess.o): in function `SSL_SESSION_free':
556 | ssl_sess.c:(.text.SSL_SESSION_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
557 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(ssl_sess.o):ssl_sess.c:(.text.SSL_SESSION_up_ref+0x14): more undefined references to `__aarch64_ldadd4_relax' follow
558 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(lhash.o): in function `getrn':
559 | lhash.c:(.text.getrn+0x3c): undefined reference to `__aarch64_ldadd8_relax'
560 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: lhash.c:(.text.getrn+0x94): undefined reference to `__aarch64_ldadd8_relax'
561 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: lhash.c:(.text.getrn+0xac): undefined reference to `__aarch64_ldadd8_relax'
562 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(lhash.o): in function `OPENSSL_LH_retrieve':
563 | lhash.c:(.text.OPENSSL_LH_retrieve+0x48): undefined reference to `__aarch64_ldadd8_relax'
564 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: lhash.c:(.text.OPENSSL_LH_retrieve+0x80): undefined reference to `__aarch64_ldadd8_relax'
565 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(drbg_lib.o): in function `RAND_DRBG_instantiate':
566 | drbg_lib.c:(.text.RAND_DRBG_instantiate+0x2d8): undefined reference to `__aarch64_ldadd4_relax'
567 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(drbg_lib.o): in function `RAND_DRBG_reseed':
568 | drbg_lib.c:(.text.RAND_DRBG_reseed+0x20c): undefined reference to `__aarch64_ldadd4_relax'
569 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(rsa_lib.o): in function `RSA_free':
570 | rsa_lib.c:(.text.RSA_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
571 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(rsa_lib.o): in function `RSA_up_ref':
572 | rsa_lib.c:(.text.RSA_up_ref+0x14): undefined reference to `__aarch64_ldadd4_relax'
573 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(threads_pthread.o): in function `CRYPTO_atomic_add':
574 | threads_pthread.c:(.text.CRYPTO_atomic_add+0x1c): undefined reference to `__aarch64_ldadd4_acq_rel'
575 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(x509_lu.o): in function `X509_STORE_free':
576 | x509_lu.c:(.text.X509_STORE_free+0x1c): undefined reference to `__aarch64_ldadd4_relax'
577 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(x509_lu.o): in function `X509_STORE_up_ref':
578 | x509_lu.c:(.text.X509_STORE_up_ref+0x14): undefined reference to `__aarch64_ldadd4_relax'
579 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(x509_set.o): in function `X509_up_ref':
580 | x509_set.c:(.text.X509_up_ref+0x14): undefined reference to `__aarch64_ldadd4_relax'
581 | /usr/lib/gcc/aarch64-alpine-linux-musl/10.2.1/../../../../aarch64-alpine-linux-musl/bin/ld: /tmp/cargo-installGDvhtd/release/deps/libopenssl_sys-b9a7b9c14ddf6366.rlib(x509cset.o): in function `X509_CRL_up_ref':
582 | x509cset.c:(.text.X509_CRL_up_ref+0x14): undefined reference to `__aarch64_ldadd4_relax'
```

Docker image `docker.io/library/rust:1.52-alpine3.13` on an arm64 machine can be used to easily reproduce.

--- a/library/std/Cargo.toml
+++ b/library/std/Cargo.toml
@@ -18,8 +18,7 @@ panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
 libc = { version = "0.2.146", default-features = false, features = ['rustc-dep-of-std'], public = true }
-# FIXME(Nilstrieb): https://github.com/rust-lang/compiler-builtins/pull/532/files#r1249354225
-compiler_builtins = { version = "=0.1.93" }
+compiler_builtins = { version = "0.1.95" }
 profiler_builtins = { path = "../profiler_builtins", optional = true }
 unwind = { path = "../unwind" }
 hashbrown = { version = "0.14", default-features = false, features = ['rustc-dep-of-std'] }

